src/neo/Network/P2P/LocalNode.cs

```diff
@@ -74,9 +74,14 @@ private void BroadcastMessage(MessageCommand command, ISerializable payload = nu
         /// Broadcast a message to all connected nodes, namely <see cref="Connections"/>.
         /// </summary>
         /// <param name="message">The message to be broadcasted.</param>
-        private void BroadcastMessage(Message message)
+        private void BroadcastMessage(Message message) => SendToRemoteNodes(message);
```

Comment:
Omg...ahauaha

Reply:
Does we need this method?

Reply:
Ping @eryeer

Reply:
I think yes, it is refered by two places. Besides, we can add some filter logics in it to filter messages in the future.